std/experimental/logger/core.d

```diff
@@ -2847,9 +2882,9 @@ private void trustedStore(T)(ref shared T dst, ref T src) @trusted
     assert(atomicOp!"=="(logged_count, 4));
 }
 
-@safe unittest
+@trusted unittest
 {
-    auto dl = cast(FileLogger) sharedLog;
+    auto dl = cast(FileLogger)cast() sharedLog;
```

Comment:
these unittests can be left as `@safe` if this cast is in an `@trusted` lambda.

Reply:
fixed